### DISCUSSION TITLE
Install hangs at thehive salt 88%

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2454] [Unsupported Versions]
ISO install v2.3.21 for Standalone on VMware hangs at thehive salt 88%. Something similar happened with ISO v2.3.10
Only thing that seems to work is using ISO v2.3.2 and then running soup to get it up to date.
Anyone else have this problem?

**Comment by defensivedepth** [comment]
Have you posted any of the logs for your previous failed installs?

Ether way, if possible, please post the `/root/so-setup.log` as well as the output of the following:

`sudo salt-call pillar.get global`
